crossterm_cursor/src/cursor/windows.rs

```diff
@@ -4,28 +4,28 @@
 
 use crossterm_utils::Result;
 
-use crate::sys::winapi::{Cursor, Handle};
+use crate::sys::windows::ScreenBufferCursor;
 
-use super::ITerminalCursor;
+use super::Cursor;
 
 /// This struct is a windows implementation for cursor related actions.
-pub struct WinApiCursor;
+pub(crate) struct WindowsCursor;
```

Comment:
I would like this to be changed back. `WinApiCursor` was named like that because I like to higlight 'winapi' . On the `AnsiCursor` i'd like to highlight that it used 'ansi'. Maybe in the future, we might have another implementation. And this name would be misleading because `AnsiCuror' works on windows as well, and my naming the cursor of this window you might get the idea that this cursor is always used by windows.

Reply:
Okay

Reply:
Fixed.